app/src/main/java/com/ryeslim/currencyexchange/MainViewModel.kt

```diff
@@ -159,4 +159,10 @@ class MainViewModel : ViewModel() {
             )
         )
     }
+
+    companion object {
+        const val eurBalanceInitial = 1000
```

Comment:
incorrectly named constants. You need to follow naming conventions: https://kotlinlang.org/docs/reference/coding-conventions.html#property-names